services/web/server/tests/unit/with_dbs/01/test_resource_manager_user_sessions.py

```diff
@@ -188,10 +189,18 @@ def get_random_int():
     await redis_registry.set_key_alive(first_key, 0)
     await redis_registry.set_key_alive(second_key, -3000)
 
-    time.sleep(1)  # minimum amount of sleep
-
-    assert await redis_registry.is_key_alive(second_key) is False
-    assert await redis_registry.is_key_alive(first_key) is False
+    async for attempt in AsyncRetrying(
+        wait=wait_fixed(0.1),
+        stop=stop_after_attempt(5),
```

Comment:
I think I would suggest to go with stop_after_delay instead of attempts. the CI machines are not monsters.

Reply:
thanks @sanderegg